src/components/backend-ai-storage-list.ts

```diff
@@ -1389,29 +1397,38 @@ export default class BackendAiStorageList extends BackendAIPage {
             @click="${(e) => this._shareFolderDialog(e)}"
             ?disabled="${this._checkProcessingStatus(rowData.item.status)}"
             style="display: ${isSharingAllowed ? '': 'none'}"
+            id="${rowData.item.id+'-share'}"
           ></mwc-icon-button>
+          <vaadin-tooltip for="${rowData.item.id+'-share'}" text="${_t('data.explorer.ShareFolder')}" position="top-start"></vaadin-tooltip>
           <mwc-icon-button
             class="fg cyan controls-running"
             icon="perm_identity"
             title=${_t('data.explorer.ModifyPermissions')}
             @click=${(e) => (this._modifyPermissionDialog(rowData.item.id))}
             ?disabled="${this._checkProcessingStatus(rowData.item.status)}"
             style="display: ${isSharingAllowed ? '': 'none'}"
+            id="${rowData.item.id+'-modifypermission'}"
           ></mwc-icon-button>
+          <vaadin-tooltip for="${rowData.item.id+'-modifypermission'}" text="${_t('data.explorer.ModifyPermissions')}" position="top-start"></vaadin-tooltip>
           <mwc-icon-button
             class="fg ${rowData.item.type == 'user' ? 'blue' : 'green'} controls-running"
             icon="create"
             title=${_t('data.folders.Rename')}
```

Comment:
`button` already has a `title` attribute. If you want to use a tooltip, let's remove the title properties.

Move your mouse cursor on the 'rename' button and wait some seconds, you can see two tooltip:
<img width="170" alt="image" src="https://github.com/lablup/backend.ai-webui/assets/621215/b8898b29-2b3a-4b80-b95c-7a0d16f3f3fa">

Reply:
Fixed.